app/src/main/java/ch/sdp/vibester/activity/BuzzerScreenActivity.kt

```diff
@@ -57,16 +56,14 @@ class GamescreenActivity : AppCompatActivity() {
         val playersFull = getIntent?.getStringArray("Player Names")
         val players = nPlayers?.let { playersFull?.copyOfRange(0, it) }
 
-        val updater = BuzzerScoreUpdater(buzIds, allPoints)
-
-        val buzToScoreMap = fetchBuzToScoreRowMap()
+        val updater = BuzzerScoreUpdater(buzIds, arrayListOf(*allPoints))
 

```

Comment:
Maybe change the declaration of allPoints to an ArrayList if possible, it might be simpler than declaring an array and then casting it.

Reply:
Fixed!